plugins/woocommerce/includes/wc-formatting-functions.php

```diff
@@ -1288,7 +1288,27 @@ function wc_format_stock_quantity_for_display( $stock_quantity, $product ) {
  * @return string
  */
 function wc_format_sale_price( $regular_price, $sale_price ) {
-	$price = '<del aria-hidden="true">' . ( is_numeric( $regular_price ) ? wc_price( $regular_price ) : $regular_price ) . '</del> <ins>' . ( is_numeric( $sale_price ) ? wc_price( $sale_price ) : $sale_price ) . '</ins>';
+	
+	// format the prices 
+	$formatted_regular_price = is_numeric( $regular_price ) ? wc_price( $regular_price ) : $regular_price;
+	$formatted_sale_price = is_numeric( $sale_price ) ? wc_price( $sale_price ) : $sale_price;
+
+	// strike through pricing
+	$price = '<del aria-hidden="true">' . $formatted_regular_price . '</del> ';
+
+	// for accessibility (a11y) we'll also display that information to screen readers
+	$price .= '<span class="screen-reader-text">';
+	$price .= sprintf( esc_html__( 'Original price was: %s.', 'woocommerce' ), $formatted_regular_price );
```

Comment:
We should probably strip HTML tags in `$formatted_regular_price` so that the end result in the `screen-reader-text` span is just plain text.

This is one way to do it, for example:

```suggestion
	$price .= esc_html( sprintf( __( 'Original price was: %s.', 'woocommerce' ), wp_strip_all_tags( $formatted_regular_price ) ) );
```

Notice that I've reversed the `esc_html()` and `sprintf()`. This is intentional because if translations accidentally include some HTML, that could break things.

Reply:
Great idea! Appreciate your thoroughness!

Added in the commit below.